import.py

```diff
@@ -40,13 +41,22 @@ def main():
 
   first = True
   for f in os.listdir(dataDir):
-    if f[-4:].lower() == ".shp":
+    rasterFound = False
+    shapefileFound = False
+    if f[-4:].lower() == ".tif":
```

Comment:
tiny nitpick: you could define 'stem' and 'extension' before this if statement to avoid a tiny bit of code duplication

so like line 46 could be
```
stem = f[:-4].replace(".", "_").replace("-","_")
extension = f[-4:].lower()

if extension == ".tif":
 ```

Reply:
I'm doing this now, but it may have to change back in future if we ever support data formats with different-length extensions, e.g. .json or .geojson .